iroha_client/tests/tx_history.rs

```diff
@@ -0,0 +1,106 @@
+#[cfg(test)]
+mod tests {
```

Comment:
Btw may be we should start removing or at least not creating new tests in such style:
```rust
#[cfg(test)]
mod tests {
    ...
}
```

It is just useless ident as for binaries in `tests/` directory